tests/fixtures/custom/custom-tsoa-template.ts.hbs

```diff
@@ -18,9 +18,11 @@ const models: TsoaRoute.Models = {
     {{#each models}}
     "{{@key}}": {
         {{#if enums}}
```

Comment:
Can we use the new check here?

Reply:
I understand handlebars so poorly that I'm hoping to not have to change it again after the the fix (commit https://github.com/lukeautry/tsoa/pull/495/commits/457407cc8d5840026879471f7d32fb63169d694c). But as you can probably tell by my commit message haha (`"fixes the broken unit test. We really should refactor the handlebars templates at some point so these mistakes are caught in the typescript compiler"`)... I think it would be awesome if none of the JSON transforms for models lived in handlebars. Maybe that's a future refactor PR?

Like it would be awesome if that whole section of the template was:
```hbs
const models: TsoaRoute.Models =  {{{modelsTemplateFunction models}}};
};
```
* where `modelsTemplateFunction` is a TypeScript transformer function that does all of the messy bits that are currently in the template.

> Can we use the new check here?

In summary, "yes" but "not as part of this PR please."